index.js

```diff
@@ -7,16 +7,79 @@ function _averageRecordSize(array) {
     return Math.floor(array.reduce((accum, str) => accum + str.length, 0) / array.length);
 }
 
+// [{offset, length}] of chunks `size` assuming `delimiter` for a file with `total` size.
+function getOffsets(size, total, delimiter) {
```

Comment:
What is the purpose of this function in the context of the chunked reader? I don't see it being used by in the chunked-file-reader, and if this is intended for generating slices, I think that should be the responsibility of the slicer

Reply:
I put here for a couple of reasons:

1. I think it's something that most all clients of chunked-file-reader would need to perform.

1. The +/- of delimiter length to the offsets was something that threw me for a loop when developing the compressed file reader.  Putting here abstracts that detail away for future readers.

Reply:
Slicers can enrich the results as needed.  Example from compressed-file-reader:
```
        getOffsets(opConfig.size, total, opConfig.delimiter).forEach((offset) => {
            slices.push({
                total,
                work,
                src,
                ...offset,
            });
        });
```